conventions/src/main/kotlin/io.opentelemetry.instrumentation.javaagent-shadowing.gradle.kts

```diff
@@ -28,6 +28,7 @@ tasks.withType<ShadowJar>().configureEach {
     // relocate(OpenTelemetry API) since these classes live in the bootstrap class loader
     relocate("io.opentelemetry.api", "io.opentelemetry.javaagent.shaded.io.opentelemetry.api")
     relocate("io.opentelemetry.semconv", "io.opentelemetry.javaagent.shaded.io.opentelemetry.semconv")
+    relocate("io.opentelemetry.semconv.incubating", "io.opentelemetry.javaagent.shaded.io.opentelemetry.semconv.incubating")
```

Comment:
could you also update https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/120da53bd30f03f54b7a09313c9e747c1363673a/javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/RemappingUrlConnection.java#L35 That is used for loading extensions that refer to these classes with their original name.

Reply:
Fixed in 435d3aeb022a4ddcb3d8efa885db2205dc1c8e8c